packages/webapp/src/pages/interfaces.ts

```diff
@@ -0,0 +1,4 @@
+export interface TreasuryStats {
```

Comment:
this file can't exist in the `pages` folder. the `pages` folder is a convention for next, only routed pages should exist here, since you don't have any react component to render this "/interfaces" page it's breaking the build.